src/solvers/dg/1d/containers.jl

```diff
@@ -0,0 +1,93 @@
+
+# Container data structure (structure-of-arrays style) for DG elements
+struct ElementContainer1D{NVARS, POLYDEG} <: AbstractContainer
+  u::Array{Float64, 3}                   # [variables, i, elements]
+  u_t::Array{Float64, 3}                 # [variables, i, elements]
+  u_tmp2::Array{Float64, 3}              # [variables, i, elements]
+  u_tmp3::Array{Float64, 3}              # [variables, i, elements]
+  inverse_jacobian::Vector{Float64}      # [elements]
+  node_coordinates::Array{Float64, 3}    # [orientation, i, elements]
+  surface_ids::Matrix{Int}               # [direction, elements]
```

Comment:
I think `surface_ids` is never used anywhere (except that it is created in the constructor below). I know that this is also true for 2D and 3D, but we can just skip adding it here in 1D right now, I think. @sloede?

Reply:
Yes, good catch. @JuliaOd can you please remove the three mentions of `surface_ids` from this file?